### ISSUE TITLE
scanjs-rules/accidental_assignment not found

### ISSUE THREAD

**Issue by machadoit** [mozfreddyb/eslint-plugin-scanjs-rules#4]
Everything seems correctly set, and I send you an example with an issue from scanjs call_open_remote=true, but it seems unable to find the accidental_assignment.

```
"rules": {"scanjs-rules/accidental_assignment":[1], "scanjs-rules/call_open_remote=true":[1]}
```

```
/src/test.js
  1:1  warning  Definition for rule 'scanjs-rules/accidental_assignment' was not found  scanjs-rules/accidental_assignment
  2:1  warning  The function open with parameter remote=true can be unsafe              scanjs-rules/call_open_remote=true
```

**Comment by mozfreddyb** [comment]
@machadoit What does test.js look like?

**Comment by machadoit** [comment]
@mozfreddyb I thought that you would not need the file, because the issue is "Definition not found", either way, this is the test file I used:

```
if (foo=bar);
window.open(target.dataset.url, '_blank', 'remote=true')
```

**Comment by mozfreddyb** [comment]
Yeah, something is odd with your eslint setup. 
I don't understand how it can find the `call_open_remote=true` test when it cannot find the `accidental_assignment` test.
Are you sure they are both in the same folder? 

Try this from the the same folder your .eslintrc is in:
```ls -l scanjs-rules/accidental_assignment.js scanjs-rules/call_open_remote=true.js```

**Comment by machadoit** [comment]
Humm, what do you mean? My folder has .eslintrc and the test.js that I send you. There aren't any more folders/files.

I added scanjs with npm install -g eslint-plugin-scanjs-rules@0.1.4 , and then with scanjs-rules/call_open_remote=true on the .eslintrc it finds issues on my test.js.

Can you please try to run it yourself and make sure you can get issues for accidental_assignment.js ?

It seems that check is referenced a lot less than the other patterns, looking at https://github.com/mozfreddyb/eslint-plugin-scanjs-rules/search?utf8=%E2%9C%93&q=accidental_assignment it really seems that it is missing something to be able to be detected correctly

**Comment by mozfreddyb** [comment]
gotcha, thank you! fix in progress.

**Comment by mozfreddyb** [comment]
Thank you for reporting this and digging into the problem, @machadoit! As you see yourself, I rarely use this project myself, so this is really appreciated.

I've pushed a fix to address this and found another unused rule that was missing in the config!
The newly published version v0.1.5 to contains the patch and it should be on npm by now.

**Comment by machadoit** [comment]
Perfect! Thx 👍
